packages/blog/src/components/home/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,13 @@
+import { HeaderImage } from '../HeaderImage'
+
+export function HomePage() {
+  return (
+    <HeaderImage
+      imgSrc="https://images.unsplash.com/photo-1506477331477-33d5d8b3dc85?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=868&q=80"
+      date={new Date()}
+      categories={['travel', 'lifestyle']}
```

Comment:
Why are the categories in English and the title and subtitle in Spanish?

Reply:
Theese are all mocked data. It will be changed later with proper data